app/helpers/api/mapping_rules_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Api::MappingRulesHelper
+  def link_to_mapping_rules(service:)
+    path, data = if current_account.independent_mapping_rules_enabled?
+                   [admin_service_proxy_rules_path(service), {}]
+                 else
+                   [edit_admin_service_integration_path(service, anchor: 'mapping-rules'), { behavior: 'open-mapping-rules' }]
+                 end
+
+    link_to 'Mapping Rules', path, anchor: 'mapping-rules', data: data
```

Comment:
anchor isn't valid for independent_mapping_rules right?

Reply:
Is it possible to include `anchor` inside `data`?

Reply:
this anchor shouldn't be here. it is already present in the else condition